deployments/charts/kuma/templates/cp-deployment.yaml

```
@@ -174,6 +182,26 @@ spec:
               readOnly: {{ $extraSecret.readOnly }}
           {{- end }}
       volumes:
+        {{- if (hasKey .Values.controlPlane "automountServiceAccountToken") }} {{- if (eq .Values.controlPlane.automountServiceAccountToken false) }}
```

With the point down there this would become:
```suggestion
        {{- if (not .Values.controlPlane.automountServiceAccountToken) }}
```